app/featureShowcase/webapp/manifest.json

```diff
@@ -119,7 +120,7 @@
                                 }
                             },
                             "initialLoad": true,
-                            "views": {
+                            "XviewsX": {
```

Comment:
This should not be here, but having both active at the same time crashes the app.